config/default/app_settings.json

```diff
@@ -209,6 +209,7 @@
       "chw"
     ],
     "can_view_last_visited_date": [],
+    "can_view_uhc_stats": [],
```

Comment:
Also add this permission to standard config.

Reply:
Done

Reply:
Done